CHANGES.rst

```diff
@@ -6,7 +6,7 @@ Changes
 .. - ...
 
 
-0.0.1
---------
+1.0.0
```

Comment:
There is already 1.1.0 on PyPI

Reply:
Alright, I switch to 2.0.0 as we change license.